libexec/scoop-status.ps1

```diff
@@ -98,8 +98,12 @@ if($missing_deps) {
     }
 }
 
-if(!$old -and !$removed -and !$failed -and !$missing_deps) {
+if(!$old -and !$removed -and !$failed -and !$missing_deps -and !$needs_update) {
     success "Everything is ok!"
 }
 
+if ($needs_update) {
+    warn "Cannot check for app updates while Scoop is out of date."
```

Comment:
It's what I'm seeing, though. Whenever there is a Scoop update *and* an update to an installed package, it will only show me the latter when I run `scoop status` again after first updating Scoop itself. That is the confusing thing I'm trying to get fixed.